src/services/contract-service.js

```diff
@@ -229,6 +230,25 @@ class ContractService {
       transactionReceipt
     }
   }
+
+  // Convert money object to correct units for blockchain
+  moneyToUnits(money) {
+    if (money.currency === 'ETH') {
+      return Web3.utils.toWei(money.amount, 'ether')
+    } else {
+      const currency = this.currencies[money.currency]
+      // handle ERC20
+      // TODO consider using ERCStandardDetailed.decimals() (for tokens that support this) so that we don't have to track decimals ourselves
+      // https://github.com/OpenZeppelin/openzeppelin-solidity/blob/6c4c8989b399510a66d8b98ad75a0979482436d2/contracts/token/ERC20/ERC20Detailed.sol
+      const currencyDecimals = currency && currency.decimals
+      if (currencyDecimals) {
+        const scaling = BigNumber(10).exponentiatedBy(currencyDecimals)
+        return String(BigNumber(money.amount).multipliedBy(scaling))
```

Comment:
how about using the BigNumber toString() method ?

Reply:
doh 🤦‍♂️